packages/mdc-drawer/README.md

```diff
@@ -320,6 +320,7 @@ Mixin | Description
 `mdc-drawer-activated-overlay-color($color)` | Sets the overlay color of the activated drawer list item.
 `mdc-drawer-scrim-fill-color($color)` | Sets the fill color of `mdc-drawer-scrim`.
 `mdc-drawer-z-index($value)` | Sets the z index of drawer. Drawer stays on top of top app bar except for clipped variant of drawer.
+`mdc-drawer-width($width)` | Sets the width of drawer and margin space required for `mdc-drawer-app-content` in case of dismissible variant.
```

Comment:
IMO this can be a little hard to read/parse (esp. the part that is dismissible-specific).

```
Sets the width of the drawer. In the case of the dismissible variant, also sets margin required for `mdc-drawer-app-content`.
```